README.md

```diff
@@ -151,7 +142,7 @@ Example commands:
 ### Install
 
 Install this project with:
```

Comment:
- To help make your repo more professional, I suggest you include steps on how to setup your project on a local desktop i.e Install, clone, and navigate into your project. Please fix this 🙏